### PR TITLE
WIP: no longer delete nodes and let the installer do azure.conf

### LINKED ISSUE #435: Azure cloud provider settings are not set on all masters
Cloud provider configuration is only set on master1 in Azure. All masters must have the cloud-configuration done or persistent storage cannot be created and used

### PR DESCRIPTION
**Pull request by cooktheryan**
Resolves #435

### CODE REVIEW

**Comment by cooktheryan** [comment]
@glennswest I would still need someone to look at the ordering but it does the needful

```

[sysdeseng@bastion ~]$ for i in master1 master2 master3 ; do ssh $i "sudo cat /etc/origin/master/master-config.yaml | grep azure"; done
  masterPublicURL: https://sysdeseng.westus.cloudapp.azure.com:8443
  publicURL: https://sysdeseng.westus.cloudapp.azure.com:8443/console/
  - sysdeseng.westus.cloudapp.azure.com
    - /etc/azure/azure.conf
    - azure
    - /etc/azure/azure.conf
    - azure
masterPublicURL: https://sysdeseng.westus.cloudapp.azure.com:8443
  assetPublicURL: https://sysdeseng.westus.cloudapp.azure.com:8443/console/
  masterPublicURL: https://sysdeseng.westus.cloudapp.azure.com:8443
  masterURL: https://sysdeseng.westus.cloudapp.azure.com:8443
  masterPublicURL: https://sysdeseng.westus.cloudapp.azure.com:8443
  publicURL: https://sysdeseng.westus.cloudapp.azure.com:8443/console/
  - sysdeseng.westus.cloudapp.azure.com
    - /etc/azure/azure.conf
    - azure
    - /etc/azure/azure.conf
    - azure
masterPublicURL: https://sysdeseng.westus.cloudapp.azure.com:8443
  assetPublicURL: https://sysdeseng.westus.cloudapp.azure.com:8443/console/
  masterPublicURL: https://sysdeseng.westus.cloudapp.azure.com:8443
  masterURL: https://sysdeseng.westus.cloudapp.azure.com:8443
  masterPublicURL: https://sysdeseng.westus.cloudapp.azure.com:8443
  publicURL: https://sysdeseng.westus.cloudapp.azure.com:8443/console/
  - sysdeseng.westus.cloudapp.azure.com
    - /etc/azure/azure.conf
    - azure
    - /etc/azure/azure.conf
    - azure
masterPublicURL: https://sysdeseng.westus.cloudapp.azure.com:8443
  assetPublicURL: https://sysdeseng.westus.cloudapp.azure.com:8443/console/
  masterPublicURL: https://sysdeseng.westus.cloudapp.azure.com:8443
  masterURL: https://sysdeseng.westus.cloudapp.azure.com:8443
```

CC @haroldwongms

**Comment by cooktheryan** [comment]
@glennswest I will need you to test this. I can't get a clean install currently. Bad news is I trashed the resource group "sysdeseng" when attempting to redeploy. For some reason it appears the bastion will not register. So it cascades into issues


````
/home/sysdeseng/openshift-install.sh: line 3: ansible: command not found
/home/sysdeseng/openshift-install.sh: line 4: ansible-playbook: command not found
/home/sysdeseng/openshift-install.sh: line 5: ansible-playbook: command not found
/home/sysdeseng/openshift-install.sh: line 7: ansible-playbook: command not found
--2017-06-06 20:17:28--  http://master1:8443/api
Resolving master1 (master1)... 10.0.0.5
Connecting to master1 (master1)|10.0.0.5|:8443... failed: No route to host.
/home/sysdeseng/openshift-install.sh: line 10: ansible-playbook: command not found
Warning: Permanently added 'master1,10.0.0.5' (ECDSA) to the list of known hosts.
Permission denied (publickey,gssapi-keyex,gssapi-with-mic).
cp: cannot stat ‘/tmp/kube-config’: No such file or directory
cp: cannot stat ‘/tmp/kube-config’: No such file or directory
Loaded plugins: langpacks, product-id, search-disabled-repos
There are no enabled repos.
 Run "yum repolist all" to see the repos you have.
 To enable Red Hat Subscription Management repositories:
     subscription-manager repos --enable <repo>
 To enable custom repositories:
     yum-config-manager --enable <repo>
setup registry for azure
/home/sysdeseng/openshift-install.sh: line 21: oc: command not found
Setup Azure PVC
/home/sysdeseng/createvhdcontainer.sh: line 7: azure: command not found
/home/sysdeseng/createvhdcontainer.sh: line 8: azure: command not found
/home/sysdeseng/createvhdcontainer.sh: line 12: azure: command not found
/home/sysdeseng/createvhdcontainer.sh: line 7: azure: command not found
/home/sysdeseng/createvhdcontainer.sh: line 8: azure: command not found
/home/sysdeseng/createvhdcontainer.sh: line 12: azure: command not found
/home/sysdeseng/openshift-install.sh: line 26: oc: command not found
/home/sysdeseng/openshift-install.sh: line 27: oc: command not found
````

File: reference-architecture/azure-ansible/bastion.sh [hunk 1]
```diff
@@ -178,9 +178,9 @@ EOF
 
 # Create Azure Cloud Provider configuration Playbook
 
-cat > /home/${AUSERNAME}/setup-azure-master.yml <<EOF
+cat > /home/${AUSERNAME}/azure-config.yml <<EOF
 #!/usr/bin/ansible-playbook
-- hosts: masters
+- hosts: all
   gather_facts: no
   serial: 1
```
Comment (e-minguez) [inline reference-architecture/azure-ansible/bastion.sh]: I'd say to remove this too

File: reference-architecture/azure-ansible/bastion.sh [hunk 2]
```diff
@@ -190,94 +190,7 @@ cat > /home/${AUSERNAME}/setup-azure-master.yml <<EOF
     azure_conf_dir: /etc/azure
     azure_conf: "{{ azure_conf_dir }}/azure.conf"
     master_conf: /etc/origin/master/master-config.yaml
```
Comment (e-minguez) [inline reference-architecture/azure-ansible/bastion.sh]: This is not longer required

**Pull request closed without merging**